text/0000-rfc-process-improvements.md

```diff
@@ -0,0 +1,72 @@
+
+So that we can maintain forward momentum, we should do everything we can to sanely deal with ideas that people are generally in support of, but nobody is willing to champion and drive to completion, and thus sit around in discussion and in draft form creating drag on the project and spreading our focus by limiting the project's work in progress.
+
+As to #2, voting members should be able to take vacation, step away from the project for personal reasons, kick back their heells on a desert island, etc. In these cases, their needs to be support for the process not stalling.
+
+# What it is
+[what-it-is]: #what-it-is
+
+This RFC proposes an amendment to [rfc#4](https://github.com/buildpacks/rfcs/pull/7/files) to address the above issues.
+
+# How it Works
+[how-it-works]: #how-it-works
+This RFC amends [rfc#4](https://github.com/buildpacks/rfcs/pull/7/files), specifically the section that begins:
+
+> Once a pull request is opened, the RFC is now in development and the following will happen:
+
+Change:
+```diff
+-- When deemed "ready", a team member will propose a "motion for final comment period (FCP)" along with a disposition of the outcome (
++- When deemed "ready", a team member will propose a "motion for final comment period (FCP)" along with a disposition. If RCF has remained open for longer than 90 days and a team member has not moved for the proposal to be moved to FCP (deemed "ready"), the RFC will be automatically tagged with `FCP` via (github action, eg.) with a disposition to close the RFC.
+```
+
+This RFC also proposes adding a new additon to the RFC process:
+
+- Core maintainers can and should delegate a voting proxy if they are going to be absent from project activities for longer than 1 week. Their delegat can be scoped globally (they can vote on anything) or scoped locally to a subteam. The maintainer in question should designate their proxy by opening a pull request with the following text which will remain open during the time of their tenure as votiong delegate:
+> For the dates of YYYY-MM-DD to YYYY-MM-DD I designate @person to vote on my behalf on RFCs covering [global, subteams]
+
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+This change gives maintainers more work and maybe cause contentious issues to either be moved through too quickly or closed prematurely.
+
+# Alternatives
+[alternatives]: #alternatives
+
+- Consider RFCs stale after 30 days of inactivity and close them without moving to FCP.
+- Automatically merge RFCs after FCP closes
+
+# Prior Art
+[prior-art]: #prior-art
+
+Not sure here
+
+# Unresolved Questions
+[unresolved-questions]: #unresolved-questions
+
+- What period of time makes sense for RFCs to be considered stale?
```

Comment:
I'd like something a bit shorter, like say 60 or 45 days, but don't have too strong an opinion on it.

Reply:
Yeah, no feelings here myself. 60 days seems like a decent place to start and we can adjust if that seems to slow or fast? I'll add language that leaves this open to future adjustment.